core/tool_roundtrip.c

```diff
@@ -567,6 +567,66 @@ _roundtrip_put_from_file (const char * const * properties)
 	oio_error_pfree (&err);
 }
 
+/* Upload a content larger than the chunk size
+ * then check the max chunk size sent */
+static void
+put_reduced_chunksize (const char * const * properties)
```

Comment:
I think you mean **increased** chunk size (the default on testing configurations is 1MiB).